data/json/itemgroups/Locations_MapExtras/locations.json

```diff
@@ -1121,7 +1121,9 @@
       { "item": "sword_xiphos", "prob": 4, "damage": [ 2, 4 ] },
       { "item": "khopesh", "prob": 4, "damage": [ 2, 4 ] },
       { "item": "spear_dory", "prob": 3, "damage": [ 2, 4 ] },
-      { "item": "pike_bronze", "prob": 3, "damage": [ 2, 4 ] }
+      { "item": "pike_bronze", "prob": 3, "damage": [ 2, 4 ] },
+      { "item": "cestus", "prob": 4, "damage": [ 2, 4 ] },
+      { "item": "bagh_nakha", "prob": 2, "damage": [ 2, 4 ] }
```

Comment:
Me: "Wait what group is this?"
museum_melee: "hehe, I'm in danger"

Reply:
The whole museum location needs an overhaul to be honest, more specialized types of museums, and more curated spawns for those specialized types.

Reply:
Ok yea I forgot the "worn out" angle, all good.